Makefile

```diff
@@ -164,7 +163,7 @@ _libhal:
 
 libnlport: $(TOOLCHAIN)/xtensa-lx106-elf/sysroot/usr/lib/libnlport.a
 
-$(TOOLCHAIN)/xtensa-lx106-elf/sysroot/usr/lib/libnlport.a: $(TOOLCHAIN)/bin/xtensa-lx106-elf-gcc
+$(TOOLCHAIN)/xtensa-lx106-elf/sysroot/usr/lib/libnlport.a: $(TOOLCHAIN)/bin/xtensa-lx106-elf-gcc sdk sdk_patch toolchain standalone
 	make -C esp_newlib_port -f ../Makefile _libnlport
 	cp -f esp_newlib_port/libnlport.a $(TOOLCHAIN)/xtensa-lx106-elf/sysroot/usr/lib/libnlport.a
 
```

Comment:
This patch should probably be folded into the  "Build procedure for libnlport.a. esp_newlib_port switched to lx106 branch".